app/views/organizations/staff/pets/tabs/_fosters.html.erb

```diff
@@ -1,24 +1,24 @@
-<div id="fosters_tab_table" class="card">
+<div id="fosters_tab_table" class="card col-6 col-xl-12">
```

Comment:
So this will be a full width table above 1200px right? Then below this it will collapse into half width and flex will place items accordingly (referring to the use of `flex-xl-row` below. 

I am curious how this looks as you drag screen width from 1200+px down. I imagine that we will show 'cards' below 1200px, but this might be a bit of a large breakpoint? What does it look like moving this breakpoint down to Large and Medium?